examples/context/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
```

Comment:
Context isn't supported yet; I don't think we want to have an example.  For now, we will want to be unit-tests only.